build/teamcity/cockroach/nightlies/stress_engflow.sh

```diff
@@ -8,7 +8,7 @@
 
 set -euo pipefail
 
-export EXTRA_TEST_ARGS="--config use_ci_timeouts"
+export EXTRA_TEST_ARGS="--config use_ci_timeoutsi --test_env=COCKROACH_NIGHTLY_STRESS=true"
```

Comment:
Also, typo in `use_ci_timeouts`.

Reply:
Oops, thats looks like vi's `s` and `i` :) Fixing.